src/python_routing_v02/compute_nhd_routing_SingleSeg_v02.py

```diff
@@ -250,6 +250,19 @@ def _handle_args():
         default="q_lateral",
     )
     parser.add_argument("--ql", help="QLat input data", dest="ql", default=None)
+
+    parser.add_argument(
+        "--data_assimilation_file_path",
```

Comment:
Change `file` --> `folder` (as in `data_assimilation_file_folder`) to be consistent with the actual usage.